tox.ini

```diff
@@ -80,7 +80,7 @@ commands = {toxinidir}/run-upgrade-tests.sh mysql-ceph
 [testenv:pep8]
 basepython = python3
 deps = hacking>=0.12
-commands = flake8
+commands = flake8 {posargs}
```

Comment:
sounds unrelated to this pr

Reply:
Removed, will do in a separate PR instead.